src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests_Preprocessor.cs

```diff
@@ -1282,6 +1282,33 @@ await TestAsync(code,
             Number("102"));
     }
 
+    [Theory, CombinatorialData, WorkItem("https://github.com/dotnet/roslyn/issues/75583")]
+    public async Task PP_AfterNonWhiteSpaceOnLine(TestHost testHost)
+    {
+        var code = """
+            if (#if false
+            true
+            #else
+            false
+            #endif
+            ) { }
+            """;
+
+        await TestAsync(code,
+            testHost,
+            ControlKeyword("if"),
+            Punctuation.OpenParen,
+            Keyword("true"),
+            PPKeyword("#"),
```

Comment:
confused by this.  shouldn't we just see this as one big skipped token now?

Reply:
We see the `#if false` as one skipped token. But we don't continue parsing disabled text, we act as if the directive wasn't there (because it's invalid), so the rest is parsed as if the `#if false` wasn't there.

Reply:
i see.